TicTacToe.cpp

```diff
@@ -1,6 +1,22 @@
 #include "TicTacToe.h"
+#include <time.h>
 
 TicTacToe::TicTacToe()
 {
+    srand(time(NULL));  // initialize random number generator
+    int a = rand();     // generate random number
+    int b = RAND_MAX / 2;   // benchmark for decision making
```

Comment:
I think you could fold the two lines here into the `if` condition fairly easily and comprehensibly. e.g. ``if(rand() > RAND_MAX / 2)`...